web/src/components/system/HostnamePage.tsx

```diff
@@ -93,20 +89,25 @@ export default function HostnamePage() {
           </Alert>
         )}
 
-        <Alert variant="custom" title={hostnameAlertTitle}>
-          {hasStaticHostname
-            ? _("This hostname is permanent and will not change unless manually updated.")
-            : _("This hostname is dynamic and may change after a reboot or network update.")}
-        </Alert>
+        {hasTransientHostname && (
+          <Alert variant="custom" title={transientHostnameAlertTitle}>
+            {_("This hostname is dynamic and may change after a reboot or network update.")}
```

Comment:
```suggestion
            {_("This hostname is dynamic and may change after a reboot or network update, as configured by the local network administrator.")}
```

"Network update" is quite vague for me. IIUC the transient hostname is provided by the DHCP server but I avoided that term and referred to a human.

Also, perhaps off topic for this PR, there are two sources for the hostname
1. this machine's administrator
2. LAN administrator

Both may pass on having an opinion.
If none is provided, we should apply `linux-a1b2c` (like YaST but 5 characters because innovation)
If both are provided, we may want to say
> Your name for this machine is `www` but other machines on the network know it as `dhcp47348.example.com`

Reply:
We could apply the naming suggestion but the question is if it should be persistent or not. We are currently not touching the persistent hostname but just modifying the transient hostname in case there is nothing and was mainly done to use Mutlicast DNS using the agama.local hostname resolution.

Related to:
- Original agama-hostname.service PR (have to find it...)
- https://github.com/agama-project/agama/pull/1432
- https://github.com/agama-project/agama/pull/1017

Reply:
Having a random one will require having to know the hostname each time, but will register the system with a different name, in that case we maybe should change it to be persistent or otherwise will be lost after rebooting the system.

There could be a problem if the network is configured later with DHCP and receiving a hostname after if it is set persistently which as I explained yesterday will not modify the hsotname as in case of set persistently then the transient hostname will be ignored..